Directory.Build.props

```
@@ -0,0 +1,13 @@
+<Project>
+    <PropertyGroup>
+        <LibplanetVersion>5.3.2-alpha.1</LibplanetVersion>
+
+        <!-- Fill with Libplanet's absolute path to debug with local Libplanet. -->
+        <LibplanetDirectory></LibplanetDirectory>
+    </PropertyGroup>
+
+    <PropertyGroup>
+        <UseLocalLibplanet Condition="Exists('$(LibplanetDirectory)')">true</UseLocalLibplanet>
+        <UseLocalLibplanet Condition=" '$(UseLocalLibplanet)' == '' ">false</UseLocalLibplanet>
+    </PropertyGroup>
```

```suggestion
    </PropertyGroup>
    
    <Target Name="DisplayInformation" BeforeTargets="Build">
        <Warning Condition="$(UseLocalLibplanet)" Text="Libplanet is being used as the local repository." />
    </Target>
```

The above task is to display a warning when Libplanet is used as a local repository during build.

---

About this comment, I used `Message` with `Importance="high"` attribute because `Warning` is treated as same with the warnings generated while compiling.

```xml
    <Target Name="NoticeBuildingWithLocalLibplanet" BeforeTargets="BeforeBuild;BeforeRebuild">
        <Message Importance="high" Text="Libplanet is being used as the local repository." Condition="$(UseLocalLibplanet)" />
    </Target>
```